ci/install-ci/kustomization.yaml

```diff
@@ -22,3 +22,6 @@ patchesStrategicMerge:
           containers:
             - name: manager
               imagePullPolicy: Never
+              env:
+                - name: RELATED_IMAGES_SIGN
+                  value: host.minikube.internal:5000/kmm/signimage:latest
```

Comment:
Can we extend `ci/install-ci/kustomization.yaml` instead?

Reply:
Can you be more explicit?

Reply:
Sorry. I though you have added it as an env as part of `.github/workflow/e2e.yaml` and I suggested to add the env variable to `ci/install-ci/kustomization.yaml` instead but this is exactly what you did!

You can close this thread.